Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Procast (iOS)'
+  os:
+    name: iOS
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: ""
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Procast/1.6.7 (com.podflitzer.android; build:16700; Android 10)'
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: "1.6.7"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'ProCast/2.6.7 (de.dsa.Cuts; build:1; iOS 16.5.1) Alamofire/4.9.1'
+  os:
+    name: iOS
+    version: "16.5.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: "2.6.7"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'PugpigBolt 2.29.3 (iPhone, iOS 16.1.1) on phone (model iPhone12,1)'
+  os:
+    name: iOS
+    version: "16.1.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: PugPig Bolt
+    version: "2.29.3"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone 11'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'PugpigBolt 3.8.10 (samsung, Android 13) on phone (model SM-G998U)'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: PugPig Bolt
+    version: "3.8.10"
+  device:
+    type: smartphone
+    brand: Samsung
+    model: 'Galaxy S21 Ultra 5G'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'GetPodcast 5.6.22 (iPhone; iPhone OS 16.4; de_DE)'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: radio.de
+    version: "5.6.22"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'radio.de/5.10.3.0 (Linux;Android 13) ExoPlayerLib/2.18.2'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: radio.de
+    version: "5.10.3.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'GetPodcast/5.10.5.3 (Linux; Android 13; de_DE) ExoPlayerLib/2.18.2'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: radio.de
+    version: "5.10.5.3"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'radio.fr 5.6.29 (iPhone; iPhone OS 16.6; fr_FR)'
+  os:
+    name: iOS
+    version: "16.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: radio.de
```

Comment:
radio.fr seems to be a separate iOS specific app https://apps.apple.com/fr/app/radio-fr-radio-et-podcast/id387352132 so the name should be "radio.fr"

Reply:
They are technically different, but from real user agents, it is obvious this company spams apps with different domains to various parts of the app store. They look similar and have very similar version numbers, so I lumped them. If you want I can separate them, but I think there are going to be at least seven different ones, if not more, once I really start looking.

```
radio.es 5.6.18 (iPhone; iPhone OS 16.3.1; es_MX)
radio.at 5.6.26 (iPhone; iPhone OS 16.5.1; de_DE)
radio.de/5845 CFNetwork/1240.0.4 Darwin/20.6.0
radio.es 5.6.20 (iPhone; iPhone OS 16.0.2; es_MX)
radio.es 5.6.16 (iPhone; iPhone OS 16.3; es_US)
radio.fr 5.6.7 (iPhone; iPhone OS 16.6; fr_FR)
radio.net 5.6.29 (iPhone; iPhone OS 16.1; en_CA)
radio.es 5.6.25 (iPhone; iPhone OS 16.5; es_US)
radio.de/5.8.0.5 (Linux;Android 12) ExoPlayerLib/2.17.1
radio.at/5845 CFNetwork/1406.0.4 Darwin/22.4.0
radio.de/app-5.2.3.5 (Linux;Android 13) ExoPlayerLib/2.11.1
radio.net 5.6.21 (iPhone; iPhone OS 16.1.2; en_GB)
radio.it/5943 CFNetwork/1408.0.4 Darwin/22.5.0
radio.net 5.6.20 (iPhone; iPhone OS 16.0.3; en_US)
radio.net/5771 CFNetwork/1335.0.3.1 Darwin/21.6.0
radio.net 5.6.25 (iPhone; iPhone OS 16.5; en_MA)
radio.net/5710 CFNetwork/1331.0.7 Darwin/21.4.0
radio.net/5923 CFNetwork/1335.0.3 Darwin/21.6.0
radio.fr 5.6.25 (iPhone; iPhone OS 16.5.1; fr_BE)
radio.net 5.6.24 (iPhone; iPhone OS 16.5; en_DE)
radio.fr/5806 CFNetwork/1402.0.8 Darwin/22.2.0
radio.fr 5.6.24 (iPhone; iPhone OS 16.5; fr_DE)
radio.de/5.8.5.1 (Linux;Android 12) ExoPlayerLib/2.18.1
radio.next/1.7.8 (Linux;Android 10) ExoPlayerLib/2.15.0
radio.de/5710 CFNetwork/1335.0.3 Darwin/21.6.0
radio.net 5.6.29 (iPhone; iPhone OS 16.6; en_MA)
radio.de/5.1.1.5 (Linux;Android 8.0.0) ExoPlayerLib/2.11.1
radio.de 5.6.19 (iPhone; iPhone OS 16.4; de_DE)
radio.de/5.10.4.0 (Linux; 5.1) ExoPlayerLib/2.18.2
radio.net 5.6.29 (iPad; iPhone OS 17.0; en_NG)
radio.fr/5845 CFNetwork/1220.1 Darwin/20.3.0
radio.net 5.6.24 (iPhone; iPhone OS 16.5; en_JP)
```

Reply:
my logic is simple: if user agent can be linked to a dedicated application page in any app store, it means that's a separate build of the app, and thus a different app. But it needs to be completely separate: for example `radio.es` is only accessible as `https://play.google.com/store/apps/details?id=de.radio.android&hl=es` with `&hl=es` and not present in apple store, but `radio.at` has not only iOS page at https://apps.apple.com/at/app/radio-at-radio-und-podcast/id375729349 but also separate Google page at https://play.google.com/store/apps/details?id=at.radio.android&hl=de_AT (note that it has id `at.radio.android` instead of  `de.radio.android`). I agree, that they are most likely still same as `de` and just rebranded, but they may have some differences under the hood, too, which may explain why they are not using universal user agent.
From the list you've shared, `de`, `fr`, `at`, `net`, `it` have dedicated pages, `es` does not. Was not able to find anything tangible for `next`

Reply:
I looked into this deeper, I found 10 domains. Of those, all of the country domains you can go to eg. `https://radio.de`, and they are reskinned versions of each other, all "powered by" the same german site. (at, de, dk, es, fr, it, pl, pt, se, net). They each have links to their app store entries, including es. For these, could I do the following single regex?

```
- regex: '^radio\.([a-z]{2}|net)[ /]([\d\.]+)'
  name: 'radio.$1'
  version: '$2'
```

I think radio.next may actually be https://radio-next.tv/, so I will separate that to a separate regex.

Reply:
yes, I think this should work for regex, based on what I see for some similar entries in yml files